ansible_runner/runner_config.py

```diff
@@ -170,8 +176,8 @@ def prepare(self):
                                                                                                            self.directory_isolation_path))
                 copy_tree(self.project_dir, self.directory_isolation_path, preserve_symlinks=True)
 
-        self.prepare_inventory()
         self.prepare_env()
+        self.prepare_inventory()
```

Comment:
What precipitates this inversion?

Reply:
This one was from me... 😄 Basically, `self.inventory` gets set inside of `prepare_inventory` and I kinda needed that to be set (properly) before building the command.